integration-tests/cucumber-tests-execution/src/main/java/org/opensmartgridplatform/cucumber/execution/JUnitCommandLineParseResult.java

```diff
@@ -10,8 +17,16 @@
 import org.junit.runner.Request;
 import org.junit.runner.manipulation.Filter;
 import org.junit.runners.model.InitializationError;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class JUnitCommandLineParseResult {
+
+    private static Logger LOGGER = LoggerFactory.getLogger(JUnitCommandLineParseResult.class);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make the "LOGGER" logger private static final. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1312)